Packs/Intezer/Integrations/IntezerV2/README.md

```diff
@@ -20,6 +20,7 @@
 <p>You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook. After you successfully execute a command, a DBot message appears in the War Room with the command details.</p>
 <ol>
 <li><a href="#h_6feb063f-66b1-45e5-89f0-1b6ff5142114" target="_self">Check file reputation by hash: intezer-analyze-by-hash</a></li>
+<li><a href="#h_353b4e58-45be-474d-b171-b8b803d216d0" target="_self">Check file reputation by latest report: intezer-get-analysis-result</a></li>
```

Comment:
```suggestion
<li><a href="#h_353b4e58-45be-474d-b171-b8b803d216d0" target="_self">Check file reputation by latest report: intezer-get-latest-report</a></li>
```

Reply:
Thanks for picking this up! Is there a generator for this file? although the extension is markdown, the file is HTML and I edited it manually